docs/src/reference/types.md

```diff
@@ -901,3 +901,73 @@ You can access definitions from the module using
 >>>
 >>> #(-3)
 ```
+
+# Selector
+A filter for selecting elements within the document.
+
+You can construct a selector in the following ways:
+- you can call the `[selector]` function
+- you can use a `[position]`
+- you can use an element function
+- you can use a regex
+- you can use a `[<label>]`
```

Comment:
```suggestion
- you can call the [`selector`]($func/selector) function
- you can use a [`location`]($func/locate)
- you can use an element function
- you can use a [string]($type/string) or [regular expression]($func/regex)
- you can use a [`{<label>}`]($func/label)
```

Reply:
Thanks, I wasn't too sure how to do the doc properly with links, etc.